src/endpoints/accounts.js

```diff
@@ -27,6 +28,24 @@ class AccountsEndpoint extends CRUDExtend {
       token
     )
   }
+
+  All(token = null, headers = {}) {
```

Comment:
I see we have a signature for this method in `.d.ts` so I'm just wondering did we have signature and not have a `.js` implementation by now? :)